common/content/world_content_service.cpp

```diff
@@ -139,6 +139,51 @@ bool WorldContentService::IsContentFlagEnabled(const std::string &content_flag)
 	return false;
 }
 
+/**
+ * @param content_flag
+ * @return
+ */
+bool WorldContentService::IsContentFlagDisabled(const std::string &content_flag)
+{
+	for (auto &f: GetContentFlags()) {
+		if (f.flag_name == content_flag && f.enabled == false) {
+			return true;
+		}
+	}
+
+	return false;
+}
+
+bool WorldContentService::DoEnabledFlagsPass(const char *content_flags) {
+	if(strlen(content_flags) == 0)
+		return true;
+
+	auto split_content_flags = SplitString(std::string(content_flags));
+
+	for (auto content_flag : split_content_flags) {
+		if(IsContentFlagEnabled(content_flag)) {
+			return true;
+		}
+	}
+
+	return false;
+}
+
+bool WorldContentService::DoDisabledFlagsPass(const char *content_flags) {
```

Comment:
We should use std::string passed in to this function

Reply:
I will work on changing to this. I tried this originally and was running into issues with it. I will take another stab.